scrapers/errata.py

```diff
@@ -68,7 +69,49 @@ def update_neo4j(self, advisories):
                 'type_': advisory['type'],
                 'update_date': advisory['update_date'],
                 'updated_at': advisory['updated_at']
-            })[0]
+            }
+
+            for associated_build in self.get_associated_builds(advisory['id']):
+                build = KojiBuild.nodes.get_or_none(id_=associated_build['id_'])
+                container_adv = False
+                if build:
+                    if build.__label__ == 'ContainerKojiBuild':
+                        adv = self.create_container_build_advisory(adv_params)
+                        container_adv = True
+                    else:
+                        adv = Advisory.create_or_update(adv_params)[0]
+
+                # If this is set, that means it was once part of the advisory but not anymore.
+                # This relationship needs to be deleted if it exists.
+                if associated_build['removed_index_id']:
+                    if build:
+                        adv.attached_builds.disconnect(build)
+                else:
+                    # This key shouldn't be stored in Neo4j
+                    del associated_build['removed_index_id']
+                    # Query Teiid only if the build is not present in Neo4j
+                    if build:
+                        if container_adv:
+                            is_container_build = True
+                        else:
+                            is_container_build = False
+                    else:
+                        try:
+                            attached_build = self.get_koji_build(associated_build['id_'])[0]
```

Comment:
After following my suggestion of making `get_koji_build` return the dictionary of the build or `None`, the `[0] will need to be dropped and the try except can be dropped as well.